snowflake/tests/snowflake_connector_patch/_snowflake_connector_patch/connector.py

```diff
@@ -44,7 +45,13 @@ def execute(self, query):
         match = TABLE_PATTERN.search(query)
         if match:
             table_name = match.group(1)
-            self.__data = getattr(tables, table_name, [])
+            # check the schema since some views have
+            # the same name in ACCOUNT_USAGE and ORGANIZATION_USAGE
+            table_prefix = ''
+            if self.schema == 'ORGANIZATION_USAGE':
+                table_prefix = 'ORGANIZATION_'
```

Comment:
👍 Works for me! A tiny nit would be that the comment mentioning view names might be a bit confusing with the final implementation.

Reply:
True, I removed the comment